src/main/java/jnr/constants/platform/fake/Shutdown.java

```diff
@@ -1,14 +1,14 @@
 // WARNING: This file is autogenerated. DO NOT EDIT!
-// Generated 2010-07-31 18:57:15 +1000
+// Generated 2018-09-11 17:01:29 -0500
 package jnr.constants.platform.fake;
 public enum Shutdown implements jnr.constants.Constant {
-SHUT_RD(0x0),
-SHUT_WR(0x1),
-SHUT_RDWR(0x2);
+SHUT_RD(1),
+SHUT_WR(2),
+SHUT_RDWR(3);
```

Comment:
@headius
The constants SHUT_RD, SHUT_WR, SHUT_RDWR have the value 0, 1, 2, respectively, and are defined in <sys/socket.h> since glibc-2.1.91.

I think this shouldn't be changed

Reply:
It appears various changes happened to both Shutdown.rb and the fake generation that were never re-run. I've fixed this to not use a bitmask and to use 0 as the base value, and it matches master again (minus the hex notation).